code/modules/admin/verbs/one_click_antag.dm

```diff
@@ -287,6 +288,29 @@
 		return 1
 	return 0
 
+/datum/admins/proc/makeMindflayers()
+	var/datum/game_mode/vampire/temp = new()
+
+	if(GLOB.configuration.gamemode.prevent_mindshield_antags)
+		temp.restricted_jobs += temp.protected_jobs
+
+	var/input_num = input(owner, "How many Mindflayers you want to create? Enter 0 to cancel","Amount:", 0) as num
```

Comment:
Why not do this `as num | null` with <=0 or null cancelling?

Reply:
Done!